pyiron_base/project/generic.py

```diff
@@ -1458,3 +1460,34 @@ def _update_jobs_in_old_database_format(self, job_name):
             elif db_entry_in_old_format:
                 for entry in db_entry_in_old_format:
                     self.db.item_update({"project": self.project_path}, entry["id"])
+
+
+    def packing(self, destination_path, csv_file_name='export.csv', compress=True):
```

Comment:
I guess the `destination_path` should allow a relative path, an absolute path as well as a project object. So we should test that the variable is converted correspondingly.

Reply:
I think with the addition of these new line, the destination_path can be a relative path, absolute path, or a Project() object.  
[https://github.com/pyiron/pyiron_base/blob/748f6e24949d973fea457c76159032901880727a/pyiron_base/archiving/export_archive.py#L73](https://github.com/pyiron/pyiron_base/blob/748f6e24949d973fea457c76159032901880727a/pyiron_base/archiving/export_archive.py#L73)